src/PhraseApp.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+namespace Translation\PlatformAdapter\PhraseApp;
+
+use FAPI\PhraseApp\PhraseAppClient;
+use Symfony\Component\Translation\MessageCatalogueInterface;
+use Translation\Common\Exception\StorageException;
+use Translation\Common\Model\Message;
+use Translation\Common\Storage;
+use Translation\Common\TransferableStorage;
+use Translation\SymfonyStorage\XliffConverter;
+
+/**
+ * @author Sascha-Oliver Prolic <saschaprolic@googlemail.com>
+ */
+class PhraseApp implements Storage, TransferableStorage
+{
+    /**
+     * @var PhraseAppClient
+     */
+    private $client;
+
+    /**
+     * @var string
+     */
+    private $projectId;
+
+    /**
+     * @var array
+     */
+    private $localeToIdMapping;
+
+    public function __construct(PhraseAppClient $client, string $projectId, array $localeToIdMapping)
+    {
+        $this->client = $client;
+        $this->projectId = $projectId;
+        $this->localeToIdMapping = $localeToIdMapping;
+    }
+
+    public function get($locale, $domain, $key)
+    {
+        throw new \BadMethodCallException('Not implemented');
+    }
+
+    public function create(Message $message)
+    {
+        throw new \BadMethodCallException('Not implemented');
+    }
+
+    public function update(Message $message)
+    {
+        throw new \BadMethodCallException('Not implemented');
+    }
+
+    public function delete($locale, $domain, $key)
+    {
+        throw new \BadMethodCallException('Not implemented');
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function export(MessageCatalogueInterface $catalogue)
+    {
+        $locale = $catalogue->getLocale();
+        $localeId = $this->getLocaleId($locale);
+
+        foreach ($catalogue->getDomains() as $domain) {
```

Comment:
While testing in a real application, I found out that the domains are not passed here. See: `php-translation/symfony-bundle/Service/StorageService.php:297`